src/main/webapp/content/scss/global.scss

```diff
@@ -419,6 +419,14 @@ a.result-orange:hover {
     color: orange;
 }
 
+.result-gray {
```

Comment:
- Not a good CSS class name as it describes the color, but not the meaning. We try to follow BEM (https://en.bem.info/methodology/), could e.g. be `result--late`
- Should not be in the global CSS file but in a component level CSS file